spec/lucky/cookies/processors/encryptor_spec.cr

```diff
@@ -0,0 +1,47 @@
+require "../../../spec_helper"
+
+include ContextHelper
+
+describe Lucky::Cookies::Processors::Encryptor do
+  describe ".read" do
+    it "returns a decrypted CookieJar" do
+      a_value = encryptor.encrypt("some cookie value")
+      a_value = Base64.strict_encode(a_value)
+      b_value = encryptor.encrypt("another cookie value")
+      b_value = Base64.strict_encode(b_value)
```

Comment:
I was a bit tripped out about the double assignment. What do you think about putting it on one line? Maybe with shorter text so it fits?